spec/javascript/PlayMoveOnClickSpec.js

```diff
@@ -0,0 +1,76 @@
+describe("Player selects a move from the web gui", function() {
+  var jsdom = require('jsdom').jsdom;
+  var doc = jsdom();
+  window = doc.defaultView;
+  document = window.document;
+
+  $ = require('jquery');
+  require('jasmine-jquery');
+  require('../../lib/javascript/PlayMoveOnClick');
+  var boardDisplay = require('../../lib/javascript/BoardDisplay');
+  var gameStatusDisplay = require('../../lib/javascript/GameStatusDisplay');
+
+  beforeEach(function() {
+    setFixtures('<table id="grid-table"><tr><td><a class="board-display" href="#" data-value="0" data-grid=[0,1,2,3,4,5,6,7,8]/></td></tr></table><p id="game-status"></p>');
+    window.MoveHandler.bindEvents();
+  });
+
+
+  it("on click, a post request should be sent with the selected move", function() {
```

Comment:
Having a _should_ in your test name implies that it might happen. Or not.
Do you want it to happen, though?
Enforce it in the test name already with something like `it("sends a POST request when clicked", function() {...});`

And - this is true for the other tests in here as well - they don't really read well in regards to the `it` description: _"it on click, a post request should be sent with the selected move"_. What is it, you want _it_ to do?